### ISSUE TITLE
[A11y] Android voiceover doesn't announce individual numbers when moving the cursor

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#90552]
b/200100958

## Steps to Reproduce

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Testing nested talkback app',
      theme: ThemeData(
        primaryColor: Colors.white,
      ),
      home: HomePage(),
    );
  }
}

class HomePage extends StatelessWidget {

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Testing nested talkback'),
      ),
      body: TextField(controller: TextEditingController(text: 'how are you'),),
    );
  }
}
```

1. turned on talkback and focus the textfield
2. double tap to start editing
3. use volumn keys to move the cursor

expects: the character is announced
actual: silence
## Logs

<details>
<summary>Logs</summary>

```
```

```
```

</details>

**Comment by TahaTesser** [comment]
@chunhtai 
Android voiceover doesn't announce individual **_characters_** when moving the cursor with volume keys, it looks like a duplicate of https://github.com/flutter/flutter/issues/84920

**Comment by chunhtai** [comment]
close as duplicat
